FAQ.md

```diff
@@ -142,16 +140,16 @@ Disabled applications and applications without internet permission are shown dim
 **(17) Why is NetGuard using so much memory?**
 
 It isn't, NetGuard doesn't allocate any memory, except a little for displaying the user interface elements.
-It appeared that on some Android variants the Play store connection, using almost 150 MB and needed for in-app donations,
-is incorrectly attributed to NetGuard instead to the Play store.
+It appeared that on some Android variants the Play Store connection, using almost 150 MB and needed for in-app donations,
+is incorrectly attributed to NetGuard instead to the Play Store.
```

Comment:
I don't like capitalization of all words, but apparently this is what Google does, so leave it.